server/config.go

```diff
@@ -67,7 +67,13 @@ func ParseConfig(bytes []byte) (*Config, error) {
 	}
 
 	c.Github.SetValuesFromEnv("")
-	c.Server.SetValuesFromEnv("BULLDOZER_")
+
+	serverPrefix := "BULLDOZER_"
+	if v, ok := os.LookupEnv("BULLDOZER_ENV_PREFIX"); ok {
+		serverPrefix = v
+	}
+	c.Server.SetValuesFromEnv(serverPrefix)
+
 	if v, ok := os.LookupEnv("BULLDOZER_PUSH_RESTRICTION_USER_TOKEN"); ok {
```

Comment:
```suggestion
	if v, ok := os.LookupEnv(envPrefix + "PUSH_RESTRICTION_USER_TOKEN"); ok {
```

I think we should use the custom prefix for all prefixed variables.